.changeset/stale-houses-grin.md

```diff
@@ -0,0 +1,6 @@
+---
+"@gradio/app": minor
+"gradio": minor
+---
+
+feat:Add `loadcomplete` function to `<gradio-app>`
```

Comment:
I would make this a highlight and add the code snippet from the guide. Looks really cool!